describe.py

```diff
@@ -395,19 +395,23 @@ def document_api(name, version, uri):
   """
     try:
         service = build(name, version)
+
+        http = build_http()
+        response, content = http.request(
+            uri or uritemplate.expand(
+                FLAGS.discovery_uri_template, {"api": name, "apiVersion": version}
+            )
+        )
+        if 'status' in response:
```

Comment:
Why this change?

Reply:
Good catch! This change isn't necessary. `describe.py` is only used to generate the docs located [here](https://github.com/googleapis/google-api-python-client/tree/master/docs/dyn). After taking another look at function `document_api`, it should not be retrieving discovery artifacts over the internet so I removed that code. Please take another look.

Reply:
In an effort to move this PR forward, I'm going to resolve this comment as the feedback has been addressed.